CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on you Machine:
+
+1. Python3 and pip3
+2. Docker or Podman
+3. Ansible
+   This can be installed by running `pip3 install ansible`
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<root directory>` with the actual path where you want to clone the repositories and set up your environment. 
```

Comment:
Maybe it is better to replace `<root directory>` with `<working directory>`

Reply:
and add:

And add to not privilege user permission to run docker if needed.
sudo usermod -a -G docker ${USER}
reboot

Reply:
Changed `<root directory>` to`<working directory>`
Added some links for additional info about how to configure